docs/src/modules/utils/helpers.js

```diff
@@ -31,3 +31,19 @@ export function pageToTitle(page) {
 
   return titleize(name);
 }
+
+export function getDependencies(raw) {
```

Comment:
Some unit tests for this method would be perfect.

Reply:
there are no tests for docs, should I put a helpers.spec.js next to it, (and add a npm script for testing also `docs/**`? 

test example: https://repl.it/@caub/BogusBestAnt

Reply:
> there are no tests for docs, should I put a helpers.spec.js next to it, (and add a npm script for testing also docs/**?

I think that adding the docs folder for the `"test:unit"` command is a good tradeoff. The tests won't be run in the CI.